### PR TITLE
Create migrations

### PR DESCRIPTION
**Pull request by shaqayq**
-Build project schema.

- Create and run the necessary migration files.
- Table and column names that match the ERD diagram.

### CODE REVIEW

File: db/migrate/20221018151725_add_post_counter_to_users.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddPostCounterToUsers < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :PostCounter, :integer
```
Comment (ShahierNashaat) [inline db/migrate/20221018151725_add_post_counter_to_users.rb:3]: - [ ] [OPTIONAL] Kindly notice that it is better to write the name of the column `snake_case` not `UpperCamelCase` as all the columns are like this.

```ruby
add_column :users, :posts_counter, :integer
```

File: db/migrate/20221018144622_create_posts.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.references :AuthorId, null: false, foreign_key: { to_table: :users }
+      t.string :title
+      t.text :text
+      t.integer :CommentCounter
+      t.integer :LikesCounter
```
Comment (ShahierNashaat) [inline db/migrate/20221018144622_create_posts.rb:8]: - [ ] [OPTIONAL] Also same here it is better to write the name of the column `snake_case` not `UpperCamelCase`.

```ruby
t.integer :comments_counter
t.integer :likes_counter
```

**Review by ShahierNashaat — APPROVED** [review]
Hi @shaqayq, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 

- Correct use of GitFlow ✔️
- Descriptive commit messages ✔️
- Professional looking pull request ✔️
- Your readme looks professional ✔️
- Your code is clean and well documented ✔️

# Approved :ok_hand: :ok_hand: :ok_hand:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/migrate/20221018144622_create_posts.rb (hunk 2 above)
Comment (shaqayq) [reply db/migrate/20221018144622_create_posts.rb:8]: @ShahierNashaat !
Thanks for your feedback.

**Pull request merged**